### PR TITLE
Layout changes to prefs window, added toggle status

### PR DESCRIPTION
**Pull request by humdingerb**

### CODE REVIEW

**Comment by threedeyes** [comment]
> And while I do appreciate @threedeyes making the initial import (and fixing the build under Haiku!), not sure adding him as author of most of the files makes much historical sense :-)

I don't care. I am not the author of this application ))

**Comment by humdingerb** [comment]
Man, do I hate this license attribution hunting...
@thomasthiriez is indeed the only Thomas Thiriez I could find as well. Let's hope he'll chime in here and allows us to use parts of his 20 year old code or I guess we cannot release this software. :(

A damn shame, because I did invest a bit of time to tweak the prefs panel and had plans to work on an icon and wanted to discuss using a deskbar replicant with you. Very frustrating, and another plea for developers to correctly copyright/license their code. :(

**Comment by thomasthiriez** [comment]
Yes, it is the same Thomas Thiriez, and sure, you can use my code with the MIT license. I am glad it is still useful 23 years later :)

**Comment by humdingerb** [comment]
The icon won't be a master piece, but should be an improvement to the current one. :)
Removed the .pre file, which sometimes appears when building...

File: src/makefile [hunk 1]
```diff
@@ -1,22 +1,134 @@
...
+#		you can simply specify XXX for the library. (e.g. the entry for
+#		"libtracker.so" would be "tracker")
+#
+#	-	for GCC-independent linking of standard C++ libraries, you can use
+#		$(STDCPPLIBS) instead of the raw "stdc++[.r4] [supc++]" library names.
+#
+#	- 	if your library does not follow the standard library naming scheme,
+#		you need to specify the path to the library and it's name.
+#		(e.g. for mylib.a, specify "mylib.a" or "path/mylib.a")
+LIBS =  be
```
[the remark this answers is not in the document]
Comment (humdingerb) [reply src/makefile:63]: Lets keep it in mind if it ever stops working... :)
As counter example, [Clipdinger](https://github.com/humdingerb/clipdinger/blob/master/input_device/makefile#L69) uses an input_server device and doesn't link to it either.

**Comment by thomasthiriez** [comment]
You are welcome! I am glad to see that my code, and the spirit of BeOS, are still alive after so much time, thanks to you!

**Comment by humdingerb** [comment]
Thanks!
I removed the non-working input_server restarting from uninstall.sh as well.

**Pull request merged**